### PR TITLE
Add an example for Tibco EMS in the documentation for siddhi-io-jms.

### PR DESCRIPTION
**Pull request by RukshiW**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: component/src/main/java/io/siddhi/extension/io/jms/source/JMSSource.java [hunk 1]
```diff
@@ -121,24 +121,59 @@
...
-                                "define stream inputStream (name string, age int, country string);")
+                                "define stream inputStream (name string, age int, country string);",
+                        description = "This example shows how to connect to an ActiveMQ queue and receive messages." +
+                                " Note that we are not providing properties like connection factory ."),
+                @Example(syntax = "@source(type = 'jms', destination = 'Q'," +
+                                "factory.initial = 'com.tibco.tibjms.naming.TibjmsInitialContextFactory'," +
+                                "provider.url = 'tibjmsnaming://localhost:7222'," +
+                                "connection.factory.jndi.name = 'QueueConnectionFactory'," +
+                                "connection.factory.type = 'queue', java.naming.security.principal = 'USER_NAME'," +
+                                "java.naming.security.credentials = 'PASSWORD', transport.jms.UserName = 'EMSUSER'," +
```
Comment (tishan89) [inline component/src/main/java/io/siddhi/extension/io/jms/source/JMSSource.java:146]: we don't need tarnsport.jms.username property.  Also did we test whether java.naming.security.principal is accepted by the source?

File: component/src/main/java/io/siddhi/extension/io/jms/source/JMSSource.java [hunk 2]
```diff
@@ -121,24 +121,59 @@
...
+                                "transport.jms.Password = 'EMSPASSWORD' \n" +
+                                "@map(type = 'xml'))\n" +
+                                "define stream ProductStream (productRef string, productName string, colour string," +
+                                "price double, available bool);",
+                        description = "This example shows how to receive messages from a queue in Tibco EMS server." +
+                                "Note that in order to receive messages from the Tibco server, you need to download" +
+                                " the Tibco JAR from the Tibco site " +
+                                "(https://www.tibco.com/products/tibco-messaging/downloads) and deploy them. e.g., " +
+                                "If you are trying this out in WSO2 Stream Processor, you need to download" +
+                                " this jar, convert it into an OSGi bundle and deploy the converted file in the " +
```
Comment (tishan89) [inline component/src/main/java/io/siddhi/extension/io/jms/source/JMSSource.java:156]: Let's add a hyperlink to this

File: component/src/main/java/io/siddhi/extension/io/jms/source/JMSSource.java [hunk 3]
```diff
@@ -121,24 +121,59 @@
...
+                                "   <productRef>CGR11GB84</productRef>\n" +
+                                "   <productName>Train</productName>\n" +
+                                "   <colour>red</colour>\n" +
+                                "   <price>16.64</price>\n" +
+                                "   <available>true</available>\n" +
+                                "</product>\n" +
+                                "In order to receive this XML message, the 'ProductStream' input stream defines a " +
+                                "matching schema with the same attributes, and '@map' annotation specifies 'XML' as" +
+                                " the mapping type. Here the connection factory type is a queue. This queue is named" +
+                                "'QueueConnectionFactory'. The 'tibjmsnaming://localhost:7222' provider URL " +
```
Comment (tishan89) [inline component/src/main/java/io/siddhi/extension/io/jms/source/JMSSource.java:170]: QueueConnectionFactory is the connection factory jndi name. not queue name

**Review by tishan89 — CHANGES_REQUESTED** [review]

File: component/src/main/java/io/siddhi/extension/io/jms/source/JMSSource.java (hunk 3 above)
Comment (nelsonandredias) [reply component/src/main/java/io/siddhi/extension/io/jms/source/JMSSource.java:170]: Hey, i am trying to replicate this exercise but i am getting the following error once i run the siddhi application: **"triggering shutdown process. javax/jms/JMSContext (Encoded)"**. Btw, the only thing i did before was pasting the OSGi converted tibjms_1.0.0.jar to <SP_HOME>/lib and the original tibjms.jar to <SP_HOME>/samples/sample-clients/lib. Am i missing something? tks in advance for all the feedback

**Pull request closed without merging**